packages/neondb/src/lib/utils/format.ts

```diff
@@ -1,3 +1,5 @@
+import { log } from "@clack/prompts";
```

Comment:
This is not needed, right?

Reply:
No 🙈 

ffs, I think I'm going to add [knip](https://knip.dev) after this PR